inabox/deploy/deploy.go

```diff
@@ -119,19 +144,44 @@ func (env *Config) deployEigenDAContracts() {
 	if err != nil {
 		log.Panicf("Error: %s", err.Error())
 	}
-	execForgeScript("script/MockRollupDeployer.s.sol:MockRollupDeployer", env.Pks.EcdsaMap[deployer.Name].PrivateKey, deployer, []string{"--sig", "run(address)", env.EigenDA.ServiceManager})
 
-	//add rollup address to path
-	data = readFile("script/output/mock_rollup_deploy_output.json")
-	var rollupAddr struct{ MockRollup string }
-	err = json.Unmarshal(data, &rollupAddr)
+	logger, err := common.NewLogger(common.DefaultLoggerConfig())
+	if err != nil {
+		log.Panicf("Error: %s", err.Error())
+	}
+
+	ethClient, err := geth.NewClient(geth.EthClientConfig{
+		RPCURLs: 		[]string{deployer.RPC},
+		PrivateKeyString: env.Pks.EcdsaMap[deployer.Name].PrivateKey[2:],
+		NumConfirmations: 0,
+		NumRetries:       0,
+	}, gcommon.Address{}, 0, logger)
+	if err != nil {
+		log.Panicf("Error: %s", err.Error())
+	}
+
+	certVerifierV1DeployCfg := env.generateV1CertVerifierDeployConfig(ethClient)
+	data, err = json.Marshal(&certVerifierV1DeployCfg)
+	if err != nil {
+		log.Panicf("Error: %s", err.Error())
+	}
+
+	// NOTE: this is pretty janky and is a short-term solution until V1 contract usage
+	//       can be deprecated.
+	writeFile("script/deploy/certverifier/config/inabox_deploy_config_v1.json", data)
+	execForgeScript("script/deploy/certverifier/CertVerifierDeployerV1.s.sol:CertVerifierDeployerV1", env.Pks.EcdsaMap[deployer.Name].PrivateKey, deployer, []string{"--sig", "run(string, string)", "inabox_deploy_config_v1.json", "inabox_v1_deploy.json"})
```

Comment:
iiuc CertVerifierDeployer only needs the `ThresholdRegistry` and ServiceManager` addresses, so you're writing those out to a json file for the deploy script to read.

Is there not another json file somewhere on the filesystem which contains these 2 contract addresses (plus all others) that we could use directly? If not then approach lgtm